src/components/Contact.tsx

```diff
@@ -0,0 +1,73 @@
+import {
+  Input,
+  Center,
+  Container,
+  StackDivider,
+  FormControl,
+  FormLabel,
+  FormErrorMessage,
+  Heading,
+  Stack,
+  Button,
+  Box,
+  Link,
+  Text
+} from "@chakra-ui/react";
+import { Link as RouterLink } from "react-router-dom";
+
+const Contact = () => {
+  return (
+    <Box>
+      <Container maxW="xl">
+        <Box>
+          <Stack
+            divider={<StackDivider />}
+            direction={["column"]}
+            spacing="6px">
+            <Box>
+              <Heading size="md">Contact</Heading>
+              <Text pt="2" fontSize="sm">
+                Hi there! Thanks for visiting Olivier. If you have another
+                question (or just want to say hello!) please feel free to drop
+                us a comment.
+              </Text>
+            </Box>
+            <Box>
+              <FormControl isRequired>
+                <FormLabel>NAME</FormLabel>
```

Comment:
BLOCKING: Mockup has these labels as not in all capitals: `Name` instead of `NAME`, etc.